src/main/frontend/src/components/SeriesSalesList.js

```diff
@@ -0,0 +1,106 @@
+//
+// IMPORTANT:
+// You must update ResourceUrl.RESOURCES_VERSION each time whenever you're modified this file!
+//
+
+class SeriesSalesList extends React.PureComponent {
+	
+	constructor(props) {
+		super(props);
+		this.state = {
+			sales: [],
+			hasServerError: false,
+		};
+		this.loadSales = this.loadSales.bind(this);
+	}
+
+	componentDidMount(){
+		this.loadSales()
+	}
+
+	loadSales() {
+		this.setState({
+			hasServerError: false,
+			sales: []
+		});
+		
+		axios.get(this.props.url)
+			.then(response => {
+				const data = response.data;
+				this.setState({ sales: data });
+		
+			})
+			.catch(error => {
+				console.error(error);
+				this.setState({ hasServerError: true });
+			});
+	}
+	
+	render() {
+		return (
+			<SeriesSalesListView
+				l10n={this.props.l10n}
+				hasServerError={this.state.hasServerError}
+				sales={this.state.sales}
+			/>
+		)
+	}
+}
+
+class SeriesSalesListView extends React.PureComponent {
+	render() {
+		const { hasServerError, l10n, sales } = this.props;
+
+		return (
+			<div className="row"> 
+ 				<div className="col-sm-12"> 
+ 					<h5>{ l10n['t_who_selling_series'] || 'Who was selling/buying this series' }</h5> 
+					 <div className="row">
+						<div id="loading-series-sales-failed-msg"
+							className={ `alert alert-danger text-center col-sm-8 col-sm-offset-2 ${hasServerError ? '' : 'hidden'}` }>
+							{ l10n['t_server_error'] || 'Server error' }
+						</div>
+					</div>
+ 					<ul>
+						{ sales.map((sale, index) => (
+							<SeriesSaleItem
+								key={sale.id}
+								l10n={this.props.l10n}
+								sale={sale}
+								index={index}
```

Comment:
If in a component we always want to have 1-bases index, let's increment it here:
```suggestion
								index={index + 1}
```